migrations/versions/3c47ba103724_proposal_status.py

```diff
@@ -13,16 +13,16 @@
 from alembic import op
 import sqlalchemy as sa
 from sqlalchemy.sql import table, column
-from funnel.models import PROPOSALSTATUS
+from funnel.models import PROPOSAL_STATE
```

Comment:
Why is a migration importing from code? Make a copy of `PROPOSALSTATUS` here (and retain the old name, since it's an old migration).

Reply:
The migration needed the states dict I guess. Really old migration.  I'll do that.